plugins/09_Shadowlands/common.lua

```diff
@@ -189,6 +189,7 @@ ns.groups.GRAPPLES = Group('grapples', 'peg_bk', {defaults=ns.GROUP_HIDDEN})
 ns.groups.HYMNS = Group('hymns', 'scroll', {defaults=ns.GROUP_HIDDEN})
 ns.groups.INQUISITORS = Group('inquisitors', 3528307, {defaults=ns.GROUP_HIDDEN})
 ns.groups.MAW_LORE = Group('maw_lore', 'chest_gy')
+ns.groups.STYGIA_NEXUS = Group('stygia_nexus', 'peg_gr')
```

Comment:
The icon here should be `peg_gn` instead of `peg_gr`. This typo causes the group icon to render as the default "chest of animal toys" icon instead of the green peg icon:

![image](https://user-images.githubusercontent.com/64381982/112590919-9573af00-8dd1-11eb-9adc-87cbc270a18c.png)

I can fix that after the merge though.